core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -3321,7 +3321,7 @@ public static String getIndexServerTempPath(String tablePath, String queryId) {
         .getProperty(CarbonCommonConstants.CARBON_INDEX_SERVER_TEMP_PATH);
     if (null == tempFolderPath) {
       tempFolderPath =
-          tablePath + "/" + CarbonCommonConstants.INDEX_SERVER_TEMP_FOLDER_NAME + "/" + queryId;
+          "/tmp/" + CarbonCommonConstants.INDEX_SERVER_TEMP_FOLDER_NAME + "/" + queryId;
```

Comment:
use CarbonCommonConstants.FILE_SEPARATOR instead on "/" in the method

Reply:
done